loaders-app/src/styles/NavBar.css

```diff
@@ -0,0 +1,14 @@
+.NavBar-navBar {
+    flex-grow: 1
+}
+.NavBar-flex {
+    flex: 1
+}
+.NavBar-navLinks {
+    margin-right: 20
+}
+.NavBar-navLink {
+    font-size: 14
+}
+
```

Comment:
Did you mention that some of these styles aren't being applied?

Reply:
Yes, the following two styles are not being applied when I view the website in the browser:

.NavBar-navLinks {
    margin-right: 20
}
.NavBar-navLink {
    font-size: 14
}

I think this may have something to do with the nesting of the JSX elements?